<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1228</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_17">
      <item>
       <widget class="QFrame" name="modePortFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <widget class="QGroupBox" name="CamAndSerialState">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>串口选择</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QComboBox" name="serialCombo"/>
              </item>
              <item>
               <widget class="QLabel" name="portStateLabel">
                <property name="text">
                 <string>port state</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_19">
              <item>
               <widget class="QPushButton" name="openCamBtn">
                <property name="text">
                 <string>open camera</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="camStateLabel">
                <property name="text">
                 <string>cma state</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="modeState">
           <property name="title">
            <string>模式选择</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QRadioButton" name="manualCheck">
              <property name="text">
               <string>手动检测</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="autoCheck">
              <property name="text">
               <string>自动检测</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="controlFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QGroupBox" name="LightState">
           <property name="title">
            <string>光源状态</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QLabel" name="mainLightLabel">
                <property name="text">
                 <string>主光1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="MainSlider">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="tickPosition">
                 <enum>QSlider::TicksBothSides</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="mainLabel">
                <property name="text">
                 <string>nan</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_13">
              <item>
               <widget class="QLabel" name="mainLight2Label">
                <property name="text">
                 <string>主光2</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="MainSlider2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="tickPosition">
                 <enum>QSlider::TicksBothSides</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="main2Label">
                <property name="text">
                 <string>nan</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <widget class="QLabel" name="subLightLabel">
                <property name="text">
                 <string>辅光1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="SubSlider">
                <property name="tracking">
                 <bool>true</bool>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="tickPosition">
                 <enum>QSlider::TicksBothSides</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="subLabel">
                <property name="text">
                 <string>nan</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <widget class="QLabel" name="subLight2Label">
                <property name="text">
                 <string>辅光2</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="SubSlider2">
                <property name="tracking">
                 <bool>true</bool>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="tickPosition">
                 <enum>QSlider::TicksBothSides</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="sub2Label">
                <property name="text">
                 <string>nan</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="lightModeLabel">
                <property name="text">
                 <string>光源模式</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="allModeBtn">
                <property name="text">
                 <string>all</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="mainModeBtn">
                <property name="text">
                 <string>main</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="subModeBtn">
                <property name="text">
                 <string>sub</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="PIRState">
           <property name="title">
            <string>红外状态</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QCheckBox" name="checkPIR1">
              <property name="text">
               <string>红外1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkPIR2">
              <property name="text">
               <string>红外2</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkPIR3">
              <property name="text">
               <string>红外3</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="MotorState">
           <property name="title">
            <string>电机参数</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="PeriodLabel">
                <property name="text">
                 <string>周期</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="periodSpin">
                <property name="value">
                 <double>0.010000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="DutyRateLabel">
                <property name="text">
                 <string>占空比</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="DutyRateSpin">
                <property name="value">
                 <number>50</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QCheckBox" name="stepMoveCheck">
                <property name="text">
                 <string>固定脉冲运动</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="pulseSpin">
                <property name="maximum">
                 <number>65535</number>
                </property>
                <property name="value">
                 <number>7053</number>
                </property>
                <property name="displayIntegerBase">
                 <number>10</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QPushButton" name="forwardBtn">
                <property name="text">
                 <string>前进</string>
                </property>
                <property name="checkable">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="reverseBtn">
                <property name="text">
                 <string>后退</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="motorStopBtn">
              <property name="text">
               <string>停止电机</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="camSetState">
           <property name="title">
            <string>相机选项</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="shutterLabel">
                <property name="text">
                 <string>曝光</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="shutterSpin"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_14">
              <item>
               <widget class="QLabel" name="frameRateLabel">
                <property name="text">
                 <string>帧率</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="frameSpin"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QLabel" name="gainLabel">
                <property name="text">
                 <string>增益</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="gainSpin"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_15">
              <item>
               <widget class="QPushButton" name="manualCapBtn">
                <property name="text">
                 <string>抓取图片</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="dispFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_18">
       <item>
        <widget class="QLabel" name="currentPic">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="text">
          <string>current</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="allPic">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="text">
          <string>all</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="mainPic">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="text">
          <string>main</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="subPic">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="text">
          <string>sub</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
   <zorder>dispFrame</zorder>
   <zorder></zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1228</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
